### ISSUE TITLE
Update AVT test suite to pass new or updated AC rules

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#11444]
It seems like our AVT test suite started failing unexpectedly. This issue is to update the failing tests and re-enable the AVT job once we have coverage. Afterward, we can require the AVT status check before merging PRs.

The following tests need to be fixed:

- [x] Form - #11486
- [x] Modal - #11488
- [X] OverflowMenu - #11490
- [X] Slider - [Bug with AC checker](https://github.com/IBMa/equal-access/issues/761)
- [x] StructuredList - no error found
- [X] TextInput #11526 
- [X] Toggle - [Bug with AC checker](https://github.com/IBMa/equal-access/issues/761)
